module/zfs/zil.c

```diff
@@ -1399,9 +1390,7 @@ zil_lwb_flush_vdevs_done(zio_t *zio)
 	 */
 	lwb->lwb_buf = NULL;
 
-	ASSERT3U(lwb->lwb_issued_timestamp, >, 0);
-	zilog->zl_last_lwb_latency = (zilog->zl_last_lwb_latency * 3 +
-	    gethrtime() - lwb->lwb_issued_timestamp) / 4;
+	zilog->zl_last_lwb_latency = (zilog->zl_last_lwb_latency * 7 + t) / 8;
```

Comment:
Why the change to 7/8th instead of 3/4ths here?

Reply:
Just a feeling that 3/4 may be too fluctuating.  Can undo if you say.

Reply:
Let's leave it as 7/8ths, I was mainly just curious.